### ISSUE TITLE
Tamriel Immersion Experience

### ISSUE THREAD

**Issue by Dredd893** [loot/oblivion#1133]
hello, been a while since I was there. Thanks for all the work you've poured into this.

Planning to play with a mod named TIE, that you can find [here](https://www.afkmods.com/index.php?/files/file/555-tie-tamriel-immersion-experience/).

TIE offers the possibility to use it together with Francesco, but you are supposed to change the tags of Francesco to do so. The readme explains it better; from TIE readme:

> Once Frans and TIE are both installed, you will need to set things up for Wrye Bash to merge the contents. Set the following files up with the listed tags:
> 
> Francesco's Leveled Creatures-Items Mod.esm   {{BASH:Actors.ACBS,Actors.AIData,Actors.AIPackages,Actors.CombatStyle,Actors.DeathItem,Actors.Stats,Delev,Invent,NpcFaces,Relev,Scripts,Stats}}
> 
> Francesco's Optional New Items Add-On.esm     {{BASH:Delev,Relev}}
> 
> Francesco's Optional New Creatures Add-On.esm {{BASH:Actors.AIPackages,Delev,Factions,Invent,Relations,Relev,Scripts}}
> 
> TIE.esp                           {BASH:Actors.ACBS,Actors.AIData,Actors.DeathItem,Actors.Spells,Actors.Stats,C.RecordFlags,Delev,Factions,Invent,NpcFaces,Relations,Relev,Scripts,Stats}}
> 
> TIE Weapon + Armor Filter.esp                 {{BASH:Deactivate,Filter,NoMerge,Stats}}
> 
> If you also happened to install Francesco's Optional New Adventurers.esp, make sure that loads before TIE. It does not require tags.
> 
> In order to also get the illuminated staves that Frans gives to many of the magic using NPCs, you need to install the TIE+Frans NPC Inventory.esp file. This file must be placed after TIE.esp in your load order and should *NOT* be tagged and should *NOT* be merged. It must be activated and left alone.

Note that I don't understand why the readme doesn't simply say to tag TIE +FRANs NPC inventory.esp file with NOMERGE and to avoid tagging it further.

1) There are missing tags, I think they come from an older wrye bash version:

    - Francesco's Leveled Creatures-Items Mod.esm . You should tag it with Npcfaces
You only have: NPCeyes, NpCfacegen

    - Francesco's Optional New Creatures Add-On.esm . You should tag it with Relations:
You have only Relations.add, relation.remove, relations.change or the multiple R(Relations)

2) You are also asked to clean Francesco's Optional New Creatures Add-On.esm from 2 ITM. Should you do it? I'm a bit reluctant whenever asked to clean these old overhaul because they are relics of old era, and for these overhauls I read a lot that "you should not clean itm from overhauls, because they are INTENDED".

__Edit__: reformatted for better readability.

**Comment by sibir-ine** [comment]
> Note that I don't understand why the readme doesn't simply say to tag TIE +FRANs NPC inventory.esp file with NOMERGE and to avoid tagging it further.

It already has the `NoMerge` tag in its description. However, from some brief testing, the Bashed Patch doesn't appear to properly forward everything unless it's tagged with `Invent.Add` & `Invent.Remove`. Also, it seems to get merged just fine. @Arthmoor, is this advice for an older version of WB, or is it still not recommended to merge or add tags to it?

> There are missing tags, I think they come from an older wrye bash version

As you noted, some of those tags (e.g., `NpcFaces` & `Relations`) are deprecated by other tags we do have. You can find a list of deprecated tags & their replacements in [the Wrye Bash docs](https://wrye-bash.github.io/docs/Wrye%20Bash%20Advanced%20Readme.html#patch-tags). There are a few missing tags that aren't deprecated, so we can investigate to see if they are applicable. Keep in mind that some of them may already be in the mods' descriptions. LOOT will display them in gray.

> I'm a bit reluctant whenever asked to clean these old overhaul because they are relics of old era, and for these overhauls I read a lot that "you should not clean itm from overhauls, because they are INTENDED".

![image](https://user-images.githubusercontent.com/69141501/154919514-93e5289e-2f9e-4226-9d85-ee7440ef7d44.png)

They're both from `Francesco's Leveled Creatures-Items Mod.esm`, so they're most likely unintentional. As a side note, older mods may have been made before the concept of dirty edits was common knowledge, so I'd argue that they're much more likely to have unintentional ITMs than intentional. I'd also argue the same for larger mods such as overhauls as the large number of records touched in the CS means there are more chances for unintentional ITMs to sneak in while mod authors work on them.

---

There also appear to be other items that may require investigation:

> It is probably also best to leave out the Living Economy support.

> If you also happened to install Francesco's Optional New Adventurers.esp, make sure that loads before TIE. It does not require tags.

**Comment by Dredd893** [comment]
First of all, thanks for the very fast answer and kindness on explaining me this stuff.

The Nomerge tag is on the description, but the TIE +FRANs NPC.esp does not seem to have it under "bash tags", at least in my computer.
![test23e](https://user-images.githubusercontent.com/61931738/154961181-50c2ce69-8c37-4c1b-8f6c-090c2dd4906b.png)
That's why I was asking. I wonder if I should add it manually. To avoid merging I simply removed the tick while making the bashed patch ("Merge patches").

About tags. I checked the readme you've linked and:

> Relations --> I now discovered that I added Relations.remove and I should not have because "The behavior of the old Relations tag is equivalent to Relations.Add combined with Relations.Change."

> Invent --> I removed invent.remove.


> For Francesco's Leveled Creatures-Items Mod.esm . I added npchair too and I think it's fine.

**Comment by sibir-ine** [comment]
> First of all, thanks for the very fast answer and kindness on explaining me this stuff.

You're welcome!

> The Nomerge tag is on the description, but the TIE +FRANs NPC.esp does not seem to have it under "bash tags", at least in my computer.

With `TIE+Frans NPC Inventory.esp` selected, right click the _Bash Tags:_ pane & select _Automatic_.

**Comment by Dredd893** [comment]
By clicking Automatic, I get three tags on the .esp: Invent.add, invent.remove, Nomerge. 

Interesting.

**Comment by sibir-ine** [comment]
`NoMerge` is from the description & `Invent.Add` & `Invent.Remove` are from LOOT.

**Edit**: you can right click the plugin in WB & select _List Bash Tags..._ to see where they're from.

**Comment by Dredd893** [comment]
Okay - so here I guess we need Arthmoor's reply about these tags.

I'm also very very curious if Francesco's Optional New Adventurers.esp doesn't need any tags at all even today.

Also, I installed Mobs version of Francesco together with Lyrondor Combat tweaks. Hopefully Arthmoor says it's the correct way to install Francesco with TIE.

**Edit:** read now the edit! I didn't know wrye could take them from Loot without actually having to open loot and sort load order. I'm accustomed to Boss and I think that even Boss doesn't work as before either - sorting the load order via boss would change tags, but I seem to have realized that right now it doesn't anymore. 

**Edit 2**: It's also not clear whether you need the Dummy .esp named Francesco's creatures and item resources.
**Desc**: _Version: 5.0__This is a dummy file that enables the required BSA resources for the creatures/items addons to load.
If you wish to remove this esp to save a slot, you can register the BSA(s) in Oblivion.ini, by adding them to the end of SArchiveList._

I think that by using wrye bash you have made the manual registering redundant?

**Comment by Arthmoor** [comment]
It's been at least 10 years since I've looked at all of the tags on this so it's going to take time before I can figure out what's still good and what's not. The instructions were written back then with the current state of Bash in mind so chances are some things have changed that need to be accounted for.

**Comment by Dredd893** [comment]
I check this page almost daily hoping for some news, so I can start playing Oblivion. Been a while since I've done that :P

**Comment by sibir-ine** [comment]
I can take a look some time this week & run my findings by Arthmoor.

**Comment by Dredd893** [comment]
Well Sibir-ine, thanks for giving a thought about it. 

I checked a topic on Arthmoor's forum and Zilav was writing about TIE needing some updates: https://www.afkmods.com/index.php?/topic/4240-relz-tie-tamriel-immersion-experience/&do=findComment&comment=160607

Hopefully Arthmoor will find the will to come back to Oblivion and to fix the best overhaul ever released :P

**Comment by sibir-ine** [comment]
I went through about half of the records in TIE that Bash Tags operate on (7208 in all) a few weeks ago & these are the tags I've verified so far with formIDs for some examples:

- Actors.AIPackages: 00015EA1, 00015EA1, 000234BB
- Actors.CombatStyle: 00011B6A, 00055FAD, 00055FAE
- Actors.RecordFlags: 00000EE7
- EnchantmentStats: 0000A949, 0000A94A, 0000A94B
- Invent.Add: 00002E99, 000034CC, 000034CD
- Invent.Change: 000244C4, 000244C5, 000244C6
- Names: 0000920F, 00037F7F, 000B1200
- NPC.Class: 00026EB3
- NPC.Eyes: 00000FC8, 00000FC8, 00000FCB
- NPC.FaceGen: 00000FC8, 00001734, 00002E94
- NPC.Hair: 00000FCC, 00000FCC, 00000FCD
- Sound: 00000B7F, 00002DBC, 00011B5E
- Text: 0001E65A, 0001E770, 0001E771

I haven't had much time to comb through the rest, so I'll just leave these here as notes until I have time to go through the rest.

**Comment by Dredd893** [comment]
Sibir-ine, thanks for all the work you're pouring into this.

**Comment by sibir-ine** [comment]
You're welcome!

**Comment by Dredd893** [comment]
Hey, have you found a bit of time to work on this?

**Comment by sibir-ine** [comment]
Yeah, I can take a look over the next couple of days.

**Comment by Dredd893** [comment]
I know that you can (and I thank you for that), but I was wondering if Arthmoor started looking at it. 

I wanted to start a new playthrough and TIE is my preferite overhaul. I guess I will have to wait.

**Comment by Dredd893** [comment]
Been a while. Any new?

**Comment by sibir-ine** [comment]
Yeah, I was planning on actually finishing this after #311.

**Comment by sibir-ine** [comment]
Okay, I've submitted #1535. I've double-checked the tags, & everything seems fine. I've added a few newer tags & removed the NPC face tags as those changes aren't as important to the mod as the other changes. It's better to let, e.g., appearance overhaul mods win those conflicts. I've also added __Deactivate__ to `TIE+Frans NPC Inventory.esp` so that only its inventory changes are imported. I'm assuming its tagging instructions were written for a version of WB where tags caused it issues. It's been nearly a decade since the mod was last updated, & WB now appears to handle the plugin well with the correct tags (& in fact needs tags to handle it correctly). Also contrary to TIE's instructions, `Francesco's Optional New Adventurers.esp` currently needs the __Relev__ tag (which it currently has) due to issues WB has with duplicate leveled list entries. I've also set TIE to load after it as instructed, though it doesn't make a huge difference with a Bashed Patch.

Re the note about Living Economy integration, that was an option in previous versions, but it no longer exists in the current version (it actually also recommends using Enhanced Economy now).

Finally, re MOBS & Lyrondor's Combat Behavior, TIE will overwrite a lot of the weapon & ammo stats changes even with a Bashed Patch, but most of their other changes should get forwarded. You'll probably still get mixed results.

@Arthmoor, feel free to take a look at my PR in case anything looks off to you.
